<launch> 
    <node pkg="aae_traj" name="aae" type="aae.py" output="screen">
        <param name="yaw_offset" type="double" value="45.0" />
        <param name="x_offset" type="double" value="-1.0" />
        <param name="y_offset" type="double" value="-1.0" />
        <param name="z_offset" type="double" value="0.5" />
    </node>
</launch>